kratos/solving_strategies/schemes/residual_based_bdf_scheme.h

```diff
@@ -123,7 +123,7 @@ class ResidualBasedBDFScheme
     explicit ResidualBasedBDFScheme(const std::size_t Order = 2)
         :ImplicitBaseType(),
          mOrder(Order),
-         mpBDFUtility(Kratos::make_unique<TimeDiscretization::BDF>(Order))
```

Comment:
I think that it is possible to use a unique pointer if you create an auxiliary unique pointer and then set the member one value by doing a `swap` operation.

Reply:
I can do this guys if you think its better than having a shared pointer. I really do not mind. @philbucher

Reply:
Well, it's just a pointer. But in this case makes sense that it is unique (there will be only one BDF utils). Moreover a unique pointer is much cheaper.

Reply:
Done, please check it is correct @rubenzorrilla